src/features/island/bumpkin/components/Feed.tsx

```diff
@@ -69,45 +65,110 @@ export const Feed: React.FC<Props> = ({ food, onFeed }) => {
     );
   }
 
+  const feed = (amount: number) => {
+    if (!selected) return;
+
+    gameService.send("bumpkin.feed", {
+      food: selected.name,
+      amount,
+    });
+  };
+
+  const inventoryFoodCount = inventory[selected.name] ?? new Decimal(0);
+  const feedVerb = isJuice(selected.name) ? "Drink" : "Eat";
+
+  const openConfirmationModal = () => {
+    if (inventoryFoodCount.lessThanOrEqualTo(1)) {
+      feed(1);
+    } else {
+      showFeedAllModal(true);
+    }
+  };
+  const closeConfirmationModal = () => {
+    showFeedAllModal(false);
+  };
+  const noFood = inventoryFoodCount.lessThanOrEqualTo(0);
+
   return (
-    <SplitScreenView
-      panel={
-        <FeedBumpkinDetails
-          details={{
-            item: selected.name,
-          }}
-          properties={{
-            xp: new Decimal(
-              getFoodExpBoost(
-                selected,
-                state.bumpkin as Bumpkin,
-                state.collectibles
-              )
-            ),
-          }}
-          actionView={
+    <>
+      <SplitScreenView
+        panel={
+          <FeedBumpkinDetails
+            details={{
+              item: selected.name,
+            }}
+            properties={{
+              xp: new Decimal(
+                getFoodExpBoost(selected, bumpkin as Bumpkin, collectibles)
+              ),
+            }}
+            actionView={
+              <>
+                <div className="flex space-x-1 sm:space-x-0 sm:space-y-1 sm:flex-col w-full">
+                  <Button
+                    disabled={inventoryFoodCount.lessThan(1)}
+                    onClick={() => feed(1)}
+                  >
+                    {`${feedVerb} ${1}`}
```

Comment:
You can just do 

${feedVerb} 1
Instead of 
${feedVerb} ${1}

Reply:
> You can just do
> 
> ${feedVerb} 1 Instead of ${feedVerb} ${1}

Wait wut lol ok let me correct that.